### PR TITLE
Issue #PS-314 feat: Learner details popup component

### PR DESCRIPTION
**Pull request by upendraTekdi**
## Description

Issue #PS-314 feat: Learner details popup component

### Changes

List the changes done to fix a bug or introducing a new feature.

## How to test

Describe the steps required to test the changes proposed in the pull request.

Please consider using the closing keyword if the pull request is proposed to
fix an issue already created in the repository
(https://help.github.com/articles/closing-issues-using-keywords/)

## Summary by CodeRabbit

- **New Features**
  - Introduced `ProfileField` component for displaying profile data in a structured grid layout.
  - Added `LearnerDetails` component for displaying detailed learner information in a responsive modal.

### CODE REVIEW

File: src/components/learner-details.tsx [hunk 1]
```diff
@@ -0,0 +1,149 @@
...
+    width: isDesktop ? 500 : 400,
+    bgcolor: 'warning.A400',
+    p: 4,
+    textAlign: 'center',
+    height: 'auto',
+  };
+  return (
+    <div>
+      <Modal
+        open={open}
```
Comment (itsvick) [inline src/components/learner-details.tsx:53]: @upendraTekdi there is modal component already created, please try to reuse same

File: src/components/learner-details.tsx [hunk 2]
```diff
@@ -0,0 +1,149 @@
...
+              display: 'flex',
+              justifyContent: 'space-between',
+              alignItems: 'center',
+            }}
+          >
+            <Typography
+              variant="h2"
+              style={{
+                textAlign: 'left',
+                color: '#4D4639',
```
Comment (itsvick) [inline src/components/learner-details.tsx:76]: @upendraTekdi colors from theme

File: src/components/learner-details.tsx [hunk 3]
```diff
@@ -0,0 +1,149 @@
...
+              onClick={handleClose}
+              variant="outlined"
+            >
+              {t('PROFILE.CLOSE')}
+            </Button>
+            <Button
+              sx={{
+                // minWidth: '100%',
+
+                color: 'black',
```
Comment (itsvick) [inline src/components/learner-details.tsx:122]: @upendraTekdi black color from the theme

File: src/components/learner-details.tsx (hunk 1 above)
Comment (upendraTekdi) [reply src/components/learner-details.tsx:53]: There have some issue in created modal component so need to implementation for that .

Comment (upendraTekdi) [reply src/components/learner-details.tsx:53]: we need some extra button for that reusable model

**Pull request merged**